wrangler-core/src/main/java/io/cdap/directives/validation/Manifest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright © 2019 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+package io.cdap.directives.validation;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.Objects;
+
+/** Contains the Schemas Manifest for standard schemas. */
+public final class Manifest implements Serializable {
+  private final Map<String, Standard> standards;
+
+  public Manifest(Map<String, Standard> standards) {
+    this.standards = standards;
```

Comment:
safer to do `this.standards = Collections.unmodifiableMap(new HashMap<>(standards))` to ensure it isn't inappropriately mutated.

Reply:
Done.